.github/workflows/nodejs.lint.yml

```diff
@@ -0,0 +1,59 @@
+name: LINTER
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Check lint with eslint
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: yarn install --frozen-lockfile
+      - run: yarn lint
+      - run: yarn workspace @app/condo lint-i18n-translations
```

Comment:
probably, it's better to move it to condo tests. It's not for all packages and apps